lms/lmsweb/views.py

```diff
@@ -96,21 +96,28 @@ def is_safe_url(target):
     )
 
 
+def get_next_url(url_next_param: Optional[str]):
+    next_url = url_next_param
+    if not is_safe_url(next_url):
+        return fail(400, "The URL isn't safe.")
+    return redirect(next_url or url_for('main'))
```

Comment:
The next_url we get from the html is a string so it's not gonna be None. It's 'None' as a string